union/versions.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "../versions.schema.json",
+  "chain_name": "union",
+  "versions": [
+    {
+      "name": "v1.0.0",
+      "recommended_version": "v1.0.0",
+      "compatible_versions": [
+        "v1.0.0"
+      ],
+      "binaries": {
+        "linux/amd64": "https://github.com/unionlabs/union/releases/download/uniond%2Fv1.0.0/uniond-release-x86_64-linux",
+        "linux/arm64": "https://github.com/unionlabs/union/releases/download/uniond%2Fv1.0.0/uniond-release-aarch64-linux"
+      },
+      "consensus": {
+        "type": "cometbls",
```

Comment:
This doesn't follow the schema. I don't think this type is allowed

Reply:
https://github.com/cosmos/chain-registry/pull/6627